modules/tinymce/changelog.txt

```diff
@@ -1,31 +1,31 @@
     Added the ability to remove images on a failed upload using the `images_upload_handler` failure callback #TINY-6011
     Added `hasPlugin` function to the editor API to determine if a plugin exists or not #TINY-766
-    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and its toggle state to be checked #TINY-6032
+    Added new `ToggleToolbarDrawer` command and query state handler to allow the toolbar drawer to be programmatically toggled and the toggle state to be checked #TINY-6032
     Added the ability to delete and navigate HTML media elements without the media plugin #TINY-4211
     Added `fullscreen_native` setting to the fullscreen plugin to enable use of the entire monitor #TINY-6284
     Added table related oxide variables to the Style API for more granular control over table cell selection appearance #TINY-6311
-    Added new `toolbar_persist` setting to control the visibility of the inline toolbar. #TINY-4847
-    Added new APIs to allow for programmatic control of the inline toolbar visibility. #TINY-4847
+    Added new `toolbar_persist` setting to control the visibility of the inline toolbar #TINY-4847
+    Added new APIs to allow for programmatic control of the inline toolbar visibility #TINY-4847
     Added the `origin` property to the `ObjectResized` and `ObjectResizeStart` events, to specify which handle the resize was preformed on #TINY-6242
     Added new StyleSheetLoader `unload` and `unloadAll` APIs to allow loaded stylesheets to be removed #TINY-3926
     Added the `LineHeight` query command and action to the editor #TINY-4843
     Added the `lineheight` toolbar and menu items, and added `lineheight` to the default format menu #TINY-4843
     Added a new `contextmenu_avoid_overlap` setting to allow contextmenus to avoid overlapping matched nodes #TINY-6036
-    Added new `listbox` dialog UI component for rendering a `selectbox` like list that accepts nested items #TINY-2236
+    Added new `listbox` dialog UI component for rendering a `selectbox` list that accepts nested items #TINY-2236
     Added back the ability to use nested items in the `image_class_list`, `link_class_list`, `link_list`, `table_class_list`, `table_cell_class_list`, and `table_row_class_list` settings #TINY-2236
     Changed how CSS manipulates table cells when selecting multiple cells to achieve a semi-transparent selection #TINY-6311
     Changed the `target` property on fired events to use the native event target. The original target for an open shadow root can be obtained using `event.getComposedPath()` #TINY-6128
     Changed the editor to cleanup loaded CSS stylesheets when all editors using the stylesheet have been removed #TINY-3926
-    Changed `imagetools` context menu icon for accessing the `image` settings to use the same icon #TINY-4141
+    Changed `imagetools` context menu icon for accessing the `image` settings to use the `image` icon #TINY-4141
```

Comment:
"`image` settings" -> "`image` dialog"? Or something...

Reply:
changed to image dialog and dropped code notation.